### ISSUE TITLE
fine-tuning on a custom dataset with the official craft-pytorch weights

### ISSUE THREAD

**Issue by alikaz3mi** [backtime92/CRAFT-Reimplementation#81]
Hello, is it possible to fine-tune your model on a custom dataset with Craft-PyTorch released weights too?

**Comment by gl94** [comment]
Yes, I just made it using the IC15 traing code. I labeled my dataset the same format as IC15.

**Comment by tahakoudri** [comment]
Hello, gl94. Can you explain how did you manage do to it and if it works well ?

**Comment by gl94** [comment]
> Hello, gl94. Can you explain how did you manage do to it and if it works well ?

It's kinda a little bit tricky, you need to use the old branch of this repo, and change some code in the trainic15data.py and maybe some function in the dataloader to get a better model. After all this been done you'll find that craft is really hard to train. I rather recommend you using dbnet in the PPocr. It's easy to train and performs very well.

**Comment by SkygirlLuna** [comment]
@gl94 What is PPocr?
